### ISSUE TITLE
Include Disco response "reply" in message back to user

### ISSUE THREAD

**Issue by thejonanshow** [heroku/dreamhouse-disco-old#29]
When a SongRequestJob posts to Disco the response will include the reply text for a user:

```
{ "reply": "Thank you for requesting 'Bonnie Tyler - Total Eclipse of The Heart'. View your song: <url>" }
```

The SongRequestJob should send a response to the user with the Facebook Client.

**Comment by thejonanshow** [comment]
@crcastle How do we want to handle this now that these replies include the album art? https://github.com/heroku/dreamhouse-disco/issues/41

I could send one response to thank them: "Thank you for requesting 'total eclipse of the heart'! I'm going to go look it up on Spotify, I'll let you know what I find."

Then another response with the album and the Casey Kasem link: "I think I've found your song, I've added 'actual song title' by 'actual artist' to the playlist. I've also made you a custom playlist based on your suggestion: '<link to kasem>'. Visit your custom playlist for a chance to win some lovely Heroku headphones. Enjoy!"

I would send the first response right away since all I need is the text from their request. Later when you give me the track information I can build the more involved message.
